packages/core/lib/commands/console.js

```diff
@@ -18,7 +18,8 @@ const command = {
       }
     ]
   },
-  run: function(options, done) {
+  run: async function (options) {
+    const { promisify } = require("util");
```

Comment:
This one can go at the top; don't need to import `"util"` or other node libraries inside the `run()`